src/ast/passes/codegen_llvm.cpp

```diff
@@ -895,6 +895,28 @@ void CodegenLLVM::visit(Call &call)
     else
       createPrintNonMapCall(call, non_map_print_id_);
   }
+  else if (call.func == "cgroup_path")
+  {
+    auto elements = AsyncEvent::CgroupPath().asLLVMType(b_);
+    StructType *cgroup_path_struct = b_.GetStructType(call.func + "_t",
+                                                      elements,
+                                                      true);
+    AllocaInst *buf = b_.CreateAllocaBPF(cgroup_path_struct,
+                                         call.func + "_args");
+
+    // Store cgroup path event id
+    b_.CreateStore(b_.GetIntSameSize(cgroup_path_id_, elements.at(0)),
+                   b_.CreateGEP(buf, { b_.getInt64(0), b_.getInt32(0) }));
```

Comment:
Specify explicit type for `CreateGEP`, this version is deprecated on LLVM 13. See #2125.